CHANGELOG.md

```diff
@@ -50,7 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 -   Added `createNewSessionWithoutModifyingResponse`, `getSessionWithoutModifyingResponse`, `refreshSessionWithoutModifyingResponse` to the Session recipe.
```

Comment:
can you share code snippet here of how to actually use getSessionWithoutModifyingResponse and handle all the errors + success state from it?